src/controllers/popupEditorBackupController.js

```diff
@@ -64,15 +61,16 @@ const popupEditorBackupController = function (PopupEditorBackups) {
       PopupEditorBackups.find({ popupId: { $in: popupId } }, (error, popupBackup) => {
```

Comment:
Same, here. We passs a callback function, but the error of `find` will get lost. The `catch` block will only catch errors that happen inside of the callback function but not the `error` parameter from the callback function if `find` method where to error out. Also, we are updating a single entry of `PopupEditorBackups`.

There should be something like findOneAndUpdate() or findByIdAndUpdate() or you can just do findById() then update the entity and save it back again. This refactor is optional, since the case where `popupBackp` is an empty array is accounted for. But still,  the code isn't clean. and I cannot imagine what could happen if an actual error is thrown by `find` method. What would `popupBackup` look like? If it's anything different than an array, we would throw a programming error.

Something to bring up to the team if necessary.